src/modules/common/LogOperation.ts

```diff
@@ -0,0 +1,23 @@
+/*******************************************************************************
+
+    This file contain the logging operations used in stoa.
+
+    Copyright:
+        Copyright (c) 2020-2021 BOSAGORA Foundation
+        All rights reserved.
+
+    License:
+        MIT License. See LICENSE for details.
+
+*******************************************************************************/
+
+let operations = {
+    db: 'DB Operation',
+    connection: 'Connection Operation',
+    blockrecovery: 'Block Recovery Operation',
+    blocksync: "Block Sync Operation",
+    start: "Stoa Server StartUp",
+    coinmarketcap: "Coin Market Cap Operation"
```

Comment:
How about a name that fits the characteristics of the work, not the name of the site (ex, coin_market)